### PR TITLE
66 game session (backend work from fall 2021)

### PR DESCRIPTION
**Pull request by emaela**

### CODE REVIEW

**Comment by emaela** [comment]
For #66 and #74

**Comment by ShashJar** [comment]
Ideas to add to/modify data representation

- [ ] boolean(?) in round data representation to indicate whether that round is the 'test' round for that particular user

File: frontend/src/screens/pairingGameScreen.tsx [hunk 1]
```diff
@@ -34,10 +42,17 @@ export default function PairingGameScreen(props: PairingGameScreenProps) {
...
+                setUser(user);
+                fi.getRoundPairs(currentRoundId).then(words => {
+                    if (isPairingGame(user)) {
+                        setEnglishWords(durstenfeldShuffle(words.map((word, i) => word.english))); // pairing case
+                        setTurkishWords(durstenfeldShuffle(words.map((word, i) => ({ turkish: word.turkish, correctEnglishWord: word.english }))));
+                    } else {
+                        setEnglishWords(durstenfeldShuffle([words.map((word, i) => word.english)[0]])); // selecting case
+                        setTurkishWords(durstenfeldShuffle(words.map((word, i) => ({ turkish: word.turkish, correctEnglishWord: word.english }))));
+                    }
+                }).catch(console.error);
```
Comment (emaela) [inline frontend/src/screens/pairingGameScreen.tsx:55]: I think setting the Turkish words is the same for both games, so we can pull that out of the conditional statement and just do setting the english words.

**Review by emaela — COMMENTED** [review]
I left some comments on ways to simplify the changes, but behavior-wise it LGTM!

**Comment by emaela** [comment]
I opened this pull request so I can't approve it - but aside from the comments I left, it looks good to merge.

**Pull request merged**